src/wasm-lib/Cargo.toml

```diff
@@ -101,3 +101,6 @@ path = "tests/modify/main.rs"
 #[patch.crates-io]
 #kittycad-modeling-cmds = { path = "../../../modeling-api/modeling-cmds" }
 #kittycad-modeling-session = { path = "../../../modeling-api/modeling-session" }
+
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = false
```

Comment:
@jtran @adamchalmers I'll need you advice on this. I had to add this to make build:wasm work on Windows ARM. Have no idea what it means.